From 8a2cd73d1fc542d5ce92a15f3bcc7566d75c68ca Mon Sep 17 00:00:00 2001
From: "Gopal, Puunithaaraj" <puunithaaraj.gopal@intel.com>
Date: Sat, 4 May 2019 00:57:45 +0800
Subject: [PATCH 1384/1443] media: ici: Fixed bytesperline calculation for
 packed format

Fixed bytesperline calculation for packed format during front end capture

Change-Id: I8cc12dc09e52a61a232e8c78bef51d2bf88e16bf
Signed-off-by: Gopal, Puunithaaraj <puunithaaraj.gopal@intel.com>
---
 drivers/media/pci/intel/ici/ici-isys-stream.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/media/pci/intel/ici/ici-isys-stream.c b/drivers/media/pci/intel/ici/ici-isys-stream.c
index a01e135..4523ce9 100644
--- a/drivers/media/pci/intel/ici/ici-isys-stream.c
+++ b/drivers/media/pci/intel/ici/ici-isys-stream.c
@@ -1015,10 +1015,10 @@ const struct ici_isys_pixelformat
 			 mpix->ffmt.width * DIV_ROUND_UP(pfmt->bpp,
 			 BITS_PER_BYTE);
 	else
-		mpix->pfmt.plane_fmt[0].bytesperline = DIV_ROUND_UP(
-			as->line_header_length + as->line_footer_length
-			+ (unsigned int)mpix->ffmt.width * pfmt->bpp,
-			BITS_PER_BYTE);
+		mpix->pfmt.plane_fmt[0].bytesperline =
+			DIV_ROUND_UP((unsigned int)mpix->ffmt.width *
+				pfmt->bpp,
+				BITS_PER_BYTE);
 
 	mpix->pfmt.plane_fmt[0].bytesperline =
 		ALIGN(mpix->pfmt.plane_fmt[0].bytesperline,
-- 
2.7.4

